### PR TITLE
Create Standard Feature Issue.md

### PR DESCRIPTION
**Pull request by HLCharlie**
Issue template for all feature requests in all repositories. Will create separate templates for Features, Bugs, and Technical

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Standard Issue.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+## Story
+**As a user of <STDcheck, Healthlabs, UTITreatment and/or Manage>**, I need <some functionality expressed as the user of the app would express the need at a high level>.
```
Comment (Chekote) [inline .github/ISSUE_TEMPLATE/Standard Issue.md:2]: Would the actor be more specific? e.g. Customer, Sales Agent, Patient Services Agent, Administrator?

Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard Issue.md:2]: yes, the actor should be more specific. I am thinking the list would be <Patient, Call Center Agent, Administrator> not sure if we need to distinguish between the call center agent types, Sales, Patient Services, Chat, and Clinical Operations but will check with Shannon on that.

File: .github/ISSUE_TEMPLATE/Standard Issue.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- [ ] Peer Reviews complete	
+- [ ] Unit and regression tests are Passed
+- [ ] Manual Testing Complete and issues resolved
+- [ ] Automated tests created and run successfully
+- [ ] All blockers addressed
+- [ ] Acceptance criteria met
+- [ ] Deployed and verified in Production
+- [ ] Ready for PO Approval
+
+## Proposed Solution
```
Comment (johnnie502) [inline .github/ISSUE_TEMPLATE/Standard Issue.md]: Can we add a section called:
```
## Subject Domain Experts
```

So the person work on the issue knows who should consult

Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard Issue.md]: Good suggestion. Do we have a published list of SMEs anywhere so the creator can put them in?

File: .github/ISSUE_TEMPLATE/Standard Issue.md (hunk 1 above)
Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard Issue.md]: I spoke with Shannon and she prefers using Agent, Admin, or Patient for the "user" field. I will make that change.

**Pull request merged**